helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -28,113 +31,98 @@ starrocksOperator:
       cpu: 500m
       memory: 200Mi
 
-
 # deploy a starrocks cluster
 starrocksCluster:
-  # Provide a name for starprocks cluster
+  # the name of starrockscluster cluster, if not set, the chart name will be used.
   name: ""
+  # the namespace of starrockscluster cluster, if not set, the release namespace will be used.
   namespace: ""
-  # annotations for starrocks cluster
+  # annotations for starrocks cluster.
   annotations: {}
-  labels:
-    object: starrocks
   # specify the cn deploy or not.
   enabledCn: false
 
-
 # spec to deploy fe.
 starrocksFESpec:
-  # Number of replicas to deploy for a fe statefulset.
+  # number of replicas to deploy for a fe statefulset.
   replicas: 1
   image:
     # image sliced by "repository:tag"
     repository: starrocks/fe-ubuntu
     tag: 2.5.5
-
-  # add annotations for fe pods. example, if you want to config monitor for datadog, you can config the annotations. etc...
+  # add annotations for fe pods. example, if you want to config monitor for datadog, you can config the annotations.
   annotations: {}
-
-  # A special supplemental group that applies to all containers in a pod. Some volume types allow the Kubelet to change the ownership of that volume to be owned by the pod:
+  # A special supplemental group that applies to all containers in a pod. Some volume types allow the Kubelet to change
+  # the ownership of that volume to be owned by the pod.
   # Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods
   fsGroup: 0
-
   # specify the service name and port config and serviceType
   # the service type refer https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
   service:
     # the fe service type, only supported ClusterIP, NodePort, LoadBalancer, ExternalName
     type: "ClusterIP"
     # the loadBalancerIP for static ip config when the type=LoadBalancer and loadbalancerIp is not empty.
-    loadbalancerIP: ""
+    loadBalancerIP: ""
```

Comment:
this will cause backwards compatible issue. if user already configured with `loadbalancerIP`, after upgrade, the configure will be lost, right?

Reply:
ok, I will cancel the modification.